sig-multicluster/charter.md

```diff
@@ -0,0 +1,45 @@
+# SIG Multicluster Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The scope of SIG Multicluster includes:
+
+- Solving common challenges related to management of multiple Kubernetes clusters and the applications that exist therein
+- Designing, discussing, implementing, and maintaining APIs, tools, and documentation related to multi-cluster administration and application management
+
+### In scope
+
+Link to SIG section in [sigs.yaml]
+
+#### Code, Binaries and Services
+
+- Mechanisms in Kubernetes that track information about multiple clusters
+- Mechanisms in Kubernetes that place resources into multiple clusters
+
+#### Cross-cutting and Externally Facing Processes
+
+- Consult with other SIGs and the community on how the mechanisms in scope for
+  the SIG should work and integrate with other areas of the wider Kubernetes
+  ecosystem
+
+### Out of scope
+
+- Software that creates or manages the lifecycle of Kubernetes clusters
```

Comment:
Which SIG (if any) would own reconciliation between cluster-api and cluster-registry?

Reply:
I had hoped to address that by narrowly defining the scope of the SIG. I will add a note that specifically addresses the situation where another part of the community plans an integration between another piece of software and one that is in scope for the SIG.